lib/index.js

```diff
@@ -326,6 +325,49 @@
     // Changed to stroke to match other stroke functions
     this.setStrokeDash = this.setLineDash;
 
+    function lastArgIs (fn, args) {
+      if (!args || !args.length) {
+        return false;
+      }
+
+      var arg = args[args.length - 1];
+
+      return fn(arg);
+    }
+
+    function isColorString (arg) {
+      return typeof arg === 'string' && !isSpecialFill.test(arg);
+    }
+
+    function isString (arg) {
+      return typeof arg === 'string';
+    }
+
+    var lastArgIsColorString = lastArgIs.bind(this, isColorString);
+    var lastArgIsString = lastArgIs.bind(this, isString);
```

Comment:
Personally I wouldnt even bother with these aliases. It's perfectly clear below to put:

`if (lastArgMatches(isColorString, args)) {`

imo